packages/client/hmi-client/src/services/model.ts

```diff
@@ -208,6 +208,10 @@ export const getTypesFromModelParts = (model: Model) => {
 	return typeMapping;
 };
 
+export function getModelParameters(model: Model): ModelParameter[] {
+	return model?.semantics?.ode?.parameters ?? [];
+}
```

Comment:
There are some overlapping here, to get parameters there is also
- model-representation/services#getParameters `https://github.com/DARPA-ASKEM/terarium/blob/921e3ee9e74950a5bc989dd6b2ced5b981315f55/packages/client/hmi-client/src/model-representation/service.ts#L104-L114`

CC @blanchco

Reply:
I switched to use the existing function. Thanks!